README.md

```diff
@@ -8,6 +8,11 @@ The usage of the page is simple:
 - Select the correct match from the search results
 - View data on cost-of-living, quality-of-living and crime rates in the area
 
+## Planning
+### Wireframe
+
+![](/imgs/wireframe__1.svg)
```

Comment:
Can you just add an 'alt' value in the square brackets please?

Reply:
completed